docs/operators/ftpInputOperator.md

```diff
@@ -0,0 +1,54 @@
+FTP Input Operator
+=============
+
+## Operator Objective
+This operator is designed to scan a directory from an FTP server for files, read
+and split file content into tuples such as lines or blocks of bytes, and finally
+emit them on the output port for further processing by downstream operators.
+The operator extends the AbstractFileInputOperator. It overrides the
+getFSInstance() method and returns an instance of the FTPFileSystem
+(org.apache.hadoop.fs.ftp.FTPFileSystem)
+
+## Class Diagram
+![FTPInputOperator class diagram](images/ftpInputOperator/classdiagram.png)
+
+### Ports
+Because this is an input operator, there are no input ports.
+
+- output: output port on which data is emitted. A downstream operator receives tuples from the FTPInputOpertor
```

Comment:
Make the first "output" bold since it is a sort of mini-title.
The second sentence is not especially useful and can be removed.